README.md

```diff
@@ -274,6 +274,38 @@ package "runtime-lib" ( ... )
 You can check that this META works for all the 4 methods described
 above.
 
+### Building rewriter that you are currently developing
+When developing a new rewriter you are very likely to prepare a few
+tests for it. The compilation line above doesn't suit this task very well
+(because ocamlfind package with your rewriter is not yet installed) 
+and it will be more convenient to specify `.cm[x]a` with your rewriter
+manually. With this approach we sacrifice compilation speed using 
+`-ppx argument` option but it should not be critical because it's very
```

Comment:
I don't really follow this, where does the slowdown comes from?

Reply:
There were a claim that combining all PPXs into single rewriter works faster then using many -ppx compiler switches

Reply:
Indeed, however I don't see why this is related to building a rewriter this way given that you can still link external ppxlib based rewriters with the local one.